### PR TITLE
NMS-13369: Add tags as query params to /health/probe Rest API

### PR DESCRIPTION
**Pull request by jadyhou**
### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-13369

### CODE REVIEW

File: smoke-test/src/test/java/org/opennms/smoketest/minion/MinionRestIT.java [hunk 1]
```diff
@@ -80,9 +80,17 @@ public void testRestHealthServiceOnMinion() throws Exception {
         given().get("/minion/rest/health")
                 .then().assertThat()
                 .statusCode(200);
-    }
-
 
+        given().get("/minion/rest/health?tag=local")
```
Comment (RangerRick) [inline smoke-test/src/test/java/org/opennms/smoketest/minion/MinionRestIT.java:84]: These should probably check that each of these actually has at least a subset of responses that you expect, and doesn't contain results for tags you don't expect, or it's not really validating that the different tags are doing something.

Comment (jadyhou) [reply smoke-test/src/test/java/org/opennms/smoketest/minion/MinionRestIT.java:84]: Yeah. You are right. I'll add those checks.

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by RangerRick** [comment]
Ulf, she has signed an OCA, she's on the wiki... TWICE! Please, I beg of you, stop posting about this.

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

File: smoke-test/src/test/java/org/opennms/smoketest/minion/MinionRestIT.java (hunk 1 above)
Comment (jadyhou) [reply smoke-test/src/test/java/org/opennms/smoketest/minion/MinionRestIT.java]: Done.

**Review by cgorantla — APPROVED** [review]
LGTM!

**Review by RangerRick — APPROVED** [review]

**Comment by opennms-bamboo** [comment]
Welcome @jadyhou.

We would like to thank you for your contribution.

I could not find a signed OpenNMS Contributor Agreement (OCA) assigned to the username @jadyhou.
In order to maintain some order with respect to the copyright to the code,
every contributor is asked to sign a "Contributor Agreement".

The OpenNMS Contributor Agreement can be downloaded [here](https://wiki.opennms.org/wiki/images/6/6b/ContributorAgreement.pdf).
Please send a signed copy to oca@opennms.org.

If you need more information, please visit https://wiki.opennms.org/wiki/Contributor_Agreement

**Pull request merged**